ml-agents/mlagents/trainers/tests/test_trainer_controller.py

```diff
@@ -120,7 +120,6 @@ def test_start_learning_trains_forever_if_no_train_model(tf_reset_graph):
     assert tc.advance.call_count == 11
     tc._export_graph.assert_not_called()
     tc._save_model.assert_not_called()
-    env_mock.close.assert_called_once()
```

Comment:
close() isn't being called here anymore. I think it's cleaner to have it in the trainer_controller since that's what creates it in the first place.